src/components/integerfield/integerfield.ts

```diff
@@ -60,14 +58,12 @@ export class MIntegerfield extends ModulVue {
     })
     public maxLength: number;
 
-    public internalValue: number;
-
     private id: string = `mIntegerfield-${uuid.generate()}`;
 
     private onPasteIntegerfield(event: Event): void {
         let pasteContent: string = event['clipboardData'].getData('text');
         if (/^\d+$/.test(pasteContent)) {
-            if (!isFinite(this.maxLengthNumber) || isFinite(this.maxLengthNumber) && String(pasteContent).length + this.internalValue.toString().length <= this.maxLength) {
+            if (!isFinite(this.maxLengthNumber) || isFinite(this.maxLengthNumber) && String(pasteContent).length + this.as<InputManagement>().internalValue.length <= this.maxLength) {
```

Comment:
On pourrait mettre cette longue condition sur 2 lignes pour éviter le scroll horizontal.

Reply:
!isFinite(this.maxLengthNumber) || isFinite(this.maxLengthNumber) rendu là on se fou que ce soit finite ou pas :P Donc enlever ces conditions.